src/js/modules/views/home.js

```diff
@@ -1,12 +1,63 @@
+import apiCall from './api.js';
+import Pokemon from '../models/pokemon.js';
+
 const content = document.getElementById('content');
 
-const home = `
-<div>
-  Home
-</div>
-`;
+const createTag = (tagName, textContent = null, className = null) => {
+  const tag = document.createElement(tagName);
+  tag.textContent = textContent;
+  tag.className = className;
+  return tag;
+};
+
+const pokemonCard = (pokemon) => {
+  const listItem = createTag('li', null, 'card');
+  const cardImage = createTag('img', null, 'card-image');
+  cardImage.src = `${pokemon.image}`;
+  const titleDiv = createTag('div', null, null);
+  const title = createTag('h2', `${pokemon.name}`, 'card-title');
+  const likeIconContainer = createTag('span', null, 'like-icon');
+  const likeIcon = createTag('i', null, 'fa fa-user');
+  likeIconContainer.appendChild(likeIcon);
+  titleDiv.appendChild(title);
+  titleDiv.appendChild(likeIconContainer);
+  const cardSubtitle = createTag('span', `${pokemon.getLikeSentence()}`, 'card-subtitle');
+  const newLigne = createTag('br', null, null);
+  const commentButton = createTag('button', 'Comments', 'comment-button');
+
+  const rowItems = [cardImage, titleDiv, cardSubtitle, newLigne, commentButton];
+
+  for (let j = 0; j < rowItems.length; j += 1) {
+    listItem.appendChild(rowItems[j]);
+  }
```

Comment:
[optional] You can use *foreach* loop instead of *for* to follow KISS principles.
Your code will look like this.
```javaScript
rowItems.foreach(item => {
 listItem.appendChild(item);
})
```